queue_processors/autoreduction_processor/test_settings.py

```diff
@@ -14,19 +14,6 @@
 
 from utils.project.structure import get_project_root
 
-# ActiveMQ
-ACTIVEMQ = {
-    "brokers": "127.0.1.1:61613",
-    "amq_queues": ["/queue/ReductionPending"],
-    "amq_user": "admin",
-    "amq_pwd": "admin",
-    "postprocess_error": "/queue/ReductionError",
```

Comment:
Interesting that previously 2 keys mapped to the same queue (`/queue/ReductionError`)
Was this for readability reasons?

Reply:
Not sure. Most likely an oversight.